src/core/a-entity.js

```diff
@@ -241,21 +240,11 @@ var proto = Object.create(ANode.prototype, {
 
       if (this.hasLoaded) { return; }
 
-      // Attach to parent object3D.
-      this.addToParent();
-
-      // Scene load.
-      function sceneLoadCallback () { self.updateComponents(); }
-      if (this.isScene) {
-        ANode.prototype.load.call(this, sceneLoadCallback);
```

Comment:
By default, nodes will wait for all child nodes to load. Entities have a special filter 'isEntity' that makes it only wait for entities. But scenes should wait on all nodes such as <a-assets< 

By maintaining the isEntity filter below, scenes will ignore assets theoretically. Perhaps we can remove that filter altogether, that might just be for animations? Though it'd be an additional change to the PR